contracts/oracle/CollateralizationOracleWrapper.sol

```diff
@@ -177,24 +198,14 @@ contract CollateralizationOracleWrapper is Timed, ICollateralizationOracle, Core
 
         require(_validityStatus, "CollateralizationOracleWrapper: CollateralizationOracle reading is invalid");
 
-        Decimal.D256 memory _thresholdLow = Decimal.from(10_000 - deviationThresholdBasisPoints).div(10_000);
-        Decimal.D256 memory _thresholdHigh = Decimal.from(10_000 + deviationThresholdBasisPoints).div(10_000);
-
-        obsolete = paused();
```

Comment:
removed the "paused" condition for exceeded deviation threshold. Feels unnatural to include that as a criteria